source-similarity-matrix.py

```diff
@@ -130,30 +143,31 @@ def get_source_id_for_title(title, sources_df):
         # Only include suggestion if within 10% of the best match's score
         top_similarity_score = 0.0
         if sources_ranking:
-          top_similarity_score = sources_ranking[0][1]
+            top_similarity_score = sources_ranking[0][1]
         similarity_cutoff = config.SIMILARITY_CUTOFF_RATIO * top_similarity_score
-        top10_dictionary[source_id] = [{'source': get_source_id_for_title(source[0], sources_df), 'score': source[1]} for
+        top10_dictionary[source_id] = [{'source': get_source_id_for_title(source[0], sources_df), 'score': source[1]}
+                                       for
                                        source in sources_ranking[:10] if source[1] > similarity_cutoff]
         top10_dictionary_human_readable[feed] = [{'source': source[0], 'score': source[1]} for source in
                                                  sources_ranking[:10] if source[1] > similarity_cutoff]
 
     logger.info("Outputting sources similarities files")
-    with open(f'output/{config.SOURCE_SIMILARITY_T10.format(LANG_REGION=lang_region)}.json', 'w', encoding='utf-8') as f:
+    with open(f'output/{config.SOURCE_SIMILARITY_T10.format(LANG_REGION=lang_region)}.json', 'w',
+              encoding='utf-8') as f:
         json.dump(top10_dictionary, f, ensure_ascii=True)
-    with open(f'output/{config.SOURCE_SIMILARITY_T10_HR.format(LANG_REGION=lang_region)}.json', 'w', encoding='utf-8') as f:
+    with open(f'output/{config.SOURCE_SIMILARITY_T10_HR.format(LANG_REGION=lang_region)}.json', 'w',
+              encoding='utf-8') as f:
         json.dump(top10_dictionary_human_readable,
                   f, ensure_ascii=True)
     logger.info("Script has finished running.")
+    upload_source_sim_files(lang_region)
```

Comment:
above this line, we could change the log to be a little more informative, i.e. _Finished updating similarity files_.

Reply:
Done

Reply:
I would change the log to "Updated similarity file for {lang_region}"